packages/esm-service-queues-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -193,11 +194,24 @@ export function useVisitQueueEntries(currServiceName: string, locationUuid: stri
   const { visitQueueNumberAttributeUuid } = config;
 
   const apiUrl = `/ws/rest/v1/visit-queue-entry?location=${queueLocationUuid}&v=full`;
+  const billingUrl = configSchema.customBillinguRL;
+
   const { t } = useTranslation();
-  const { data, error, isLoading, isValidating, mutate } = useSWR<{ data: { results: Array<VisitQueueEntry> } }, Error>(
-    apiUrl,
-    openmrsFetch,
-  );
+  const {
+    data: queueData,
+    error,
+    isLoading,
+    isValidating,
+    mutate,
+  } = useSWR<{ data: { results: Array<VisitQueueEntry> } }, Error>(apiUrl, openmrsFetch);
+
+  const {
+    data: billsData,
+    error: billsError,
+    isLoading: billsIsLoading,
+  } = useSWR<{ data: { results: Array<any> } }>(billingUrl, openmrsFetch, {
```

Comment:
```suggestion
  } = useSWR<{ data: { results: Array<any> } }>(billingUrl ? billingUrl: null, openmrsFetch, {
```
Passing null in the key doesn't make a request